Sources/SwiftDocC/Infrastructure/Symbol Graph/AccessControl+Comparable.swift

```diff
@@ -11,18 +11,13 @@
 import SymbolKit
 
 extension SymbolGraph.Symbol.AccessControl: Comparable {
-    private var level: Int? {
+    private var level: UInt? {
```

Comment:
Should this change be in this PR? Swift generally advises against using `UInt` as a way to enforce positive values: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics/#UInt.

Reply:
No it shouldn't be in here.